lib/manager/terragrunt/readme.md

```diff
@@ -0,0 +1,9 @@
+Currently Terragrunt support is limited to Terraform registry sources and github sources that include semver refs, e.g. like `github.com/hashicorp/example?ref=v1.0.0`.
```

Comment:
I'm interested to know how many other types there are which we might want to support in future?

Reply:
I've updated documentation to include info about custom config for using ref which are not in line with semver.

The text you mentioned is basically the same as for Terraform, but just FYI all the available source types supported by both of them can be [found here](https://www.terraform.io/docs/modules/sources.html)